pom.xml

```diff
@@ -33,9 +33,15 @@
 
     <dependencies>
         <dependency>
-            <groupId>com.microsoft.azure.netapp.v2019_11_01</groupId>
-            <artifactId>azure-mgmt-netapp</artifactId>
-            <version>1.0.0-beta-2</version>
+            <groupId>com.azure</groupId>
+            <artifactId>azure-identity</artifactId>
```

Comment:
were this changed in the new version ? how is azure-identity related to NetApp ?